### PR TITLE
Add vote listing for users

### PR DESCRIPTION
**Pull request by ScriptDeBurger**

### CODE REVIEW

**Comment by Typiqally** [comment]
Fix merge conflicts please

**Comment by ScriptDeBurger** [comment]
I solved the conflicts, can you review the code?

File: src/routes/api/quotes/+server.ts [hunk 1]
```diff
@@ -46,6 +55,7 @@ export const GET: RequestHandler = async (reqEvent: RequestEvent) => {
     });
 
     return new Response(JSON.stringify({
+
```
Comment (Typiqally) [inline src/routes/api/quotes/+server.ts]: Remove this

File: src/routes/api/quotes/votes/+server.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import {error} from "@sveltejs/kit";
+
+export const GET: RequestHandler = async (reqEvent: RequestEvent) => {
+    const discordId = reqEvent.url.searchParams.get('discordId');
+
+    if (!discordId) {
+        throw error(400, 'Missing discord id.');
+    }
+
+    const getVotes = await db.vote.findMany({
```
Comment (Typiqally) [inline src/routes/api/quotes/votes/+server.ts]: votes

**Review by Typiqally — CHANGES_REQUESTED** [review]

**Review by Typiqally — APPROVED** [review]

**Pull request merged**